api/src/test/java/com/epam/pipeline/acl/docker/DockerRegistryApiServiceTest.java

```diff
@@ -0,0 +1,263 @@
+    }
+
+    @Test
+    @WithMockUser(username = SIMPLE_USER)
+    public void shouldUpdateDockerRegistryWhenPermissionIsGranted() {
+        initAclEntity(dockerRegistryWithOwner, AclPermission.WRITE);
+        doReturn(SIMPLE_USER).when(mockAuthManager).getAuthorizedUser();
+        doReturn(dockerRegistryWithOwner).when(mockDockerRegistryManager).updateDockerRegistry(dockerRegistry);
+
+        final DockerRegistry resultRegistry = dockerRegistryApiService.updateDockerRegistry(dockerRegistry);
+
+        assertThat(resultRegistry).isEqualTo(dockerRegistryWithOwner);
+    }
+
+    @Test
+    @WithMockUser
+    public void shouldDenyUpdateDockerRegistryWhenPermissionIsNotGranted() {
+        doReturn(dockerRegistryWithOwner).when(mockDockerRegistryManager).updateDockerRegistry(dockerRegistry);
+        initAclEntity(dockerRegistryWithOwner);
+        doReturn(SIMPLE_USER).when(mockAuthManager).getAuthorizedUser();
+
+        assertThrows(AccessDeniedException.class,
+            () -> dockerRegistryApiService.updateDockerRegistry(dockerRegistryWithOwner));
+    }
+
+    @Test
+    @WithMockUser(roles = ADMIN_ROLE)
+    public void shouldUpdateDockerRegistryCredentialsForAdmin() {
+        doReturn(dockerRegistry).when(mockDockerRegistryManager).updateDockerRegistryCredentials(dockerRegistryVO);
+
+        assertThat(dockerRegistryApiService.updateDockerRegistryCredentials(dockerRegistryVO))
+                .isEqualTo(dockerRegistry);
+    }
+
+    @Test
+    @WithMockUser
+    public void shouldDenyUpdateDockerRegistryCredentialsForNonAdminUser() {
+        doReturn(dockerRegistry).when(mockDockerRegistryManager).updateDockerRegistryCredentials(dockerRegistryVO);
+
+        assertThrows(AccessDeniedException.class,
+            () -> dockerRegistryApiService.updateDockerRegistryCredentials(dockerRegistryVO));
+    }
+
+    @Test
+    @WithMockUser
+    public void shouldListDockerRegistriesWithCerts() {
+        doReturn(dockerRegistryList).when(mockDockerRegistryManager).listAllDockerRegistriesWithCerts();
+
+        assertThat(dockerRegistryApiService.listDockerRegistriesWithCerts()).isEqualTo(dockerRegistryList);
+    }
+
+    @Test
+    @WithMockUser
+    public void shouldLoadAllRegistriesContent() {
+        doReturn(dockerRegistryList).when(mockDockerRegistryManager).loadAllRegistriesContent();
+
+        assertThat(dockerRegistryApiService.loadAllRegistriesContent()).isEqualTo(dockerRegistryList);
+    }
+
+    @Test
+    @WithMockUser
+    public void shouldLoadDockerRegistry() {
+        doReturn(dockerRegistry).when(mockDockerRegistryManager).load(ID);
+
+        assertThat(dockerRegistryApiService.load(ID)).isEqualTo(dockerRegistry);
+    }
+
+    @Test
+    @WithMockUser(roles = ADMIN_ROLE)
+    public void shouldDeleteDockerRegistryForAdmin() {
+        doReturn(dockerRegistry).when(mockDockerRegistryManager).delete(ID, true);
+
+        assertThat(mockDockerRegistryManager.delete(ID, true)).isEqualTo(dockerRegistry);
```

Comment:
Apparently, we should use the `dockerRegistryApiService` here, not the `mockDockerRegistryManager`. And also in the test below.